src/nni_manager/package.json

```diff
@@ -39,6 +39,7 @@
     "@types/express": "^4.16.0",
     "@types/glob": "^7.1.1",
     "@types/js-base64": "^2.3.1",
+    "@types/js-yaml": "^3.12.5",
```

Comment:
already has this `js-yaml` package? refer line 72

Reply:
It uses to provide a TypeScript type declaration, so that we can import js lib like `import * as yaml from 'js-yaml';`, instead of `const yaml = require('js-yaml');`.